src/components/icons/index.js

```diff
@@ -0,0 +1,27 @@
+import LabelIcon from '../icons/label';
```

Comment:
This is actually just a refactoring of what already existed inside `IconButton`, I just moved it out into a file so it can be reused and follow the same pattern again in `NodeListRow` for consistency.

If you're still keen to change it though, I guess it should be possible to just pass the icon component directly or as a render prop instead of a string?